packages/@webex/plugin-meetings/src/common/errors/join-status-error.ts

```diff
@@ -0,0 +1,24 @@
+import {ERROR_DICTIONARY} from '../../constants';
+
+/**
+ * Extended Error object for join flow call meetinginfo api error
+ */
+export default class JoinStatusError extends Error {
+  code: any;
```

Comment:
can we have better types than `any`?

Reply:
For the code and sdkMessage can use string.